### ISSUE TITLE
Unable to install mxnet

### ISSUE THREAD

**Issue by khoaimon180901** [apache/incubator-mxnet#19087]
I can not use 'pip install mxnet' in command prompt to intstall mxnet. It keep appearing a list with long of issues that I hardly to know what it is . Here it is a small part of my issue
![image](https://user-images.githubusercontent.com/70476363/92233211-10c74f00-eeda-11ea-9d29-8c0ee3ae4f0f.png)

**Comment by ekdnam** [comment]
For Apache MXNet, one of the requirements is OpenBLAS or MKL. You perhaps have to download and build one of them first. 

In your screenshot, in the bottom-most part, it can be seen that the prompt says something about MKL. 

Please refer this as well: https://pypi.org/project/mxnet/ . Here it is written that MXNet uses OpenBLAS.

@szha do correct if I am wrong.

**Comment by leezu** [comment]
Try `pip install --upgrade pip`.

This issue is not related to MXNet

**Comment by szha** [comment]
@ekdnam yes, MXNet does support OpenBLAS or MKL as the BLAS library, though for that package OpenBLAS is already bundled.

As @leezu mentioned, the error actually happened during installation of numpy, so @khoaimon180901 your best bet is likely to first upgrade pip and see if it can match pre-built package instead of getting matched only to source distribution (which requires building during installation)
